src/OpenTelemetry/CHANGELOG.md

```diff
@@ -42,6 +42,15 @@
   [IMetricsListener](https://learn.microsoft.com/dotNet/api/microsoft.extensions.diagnostics.metrics.imetricslistener).
   ([#5265](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5265))
 
+* **Experimental (pre-release builds only):** `Exemplar` and `ExemplarReservoir`
```

Comment:
nit:
Reservoirs were purely internal, even in experimental builds right? So this is not affecting end users.

The only changelog entry should be for the authors of custom exporters, as they see breaking changes.

Reply:
How's this? https://github.com/open-telemetry/opentelemetry-dotnet/pull/5386/commits/b10a2c9f7032e9dc990ff0acb0a4a88e99b33b67